### ISSUE TITLE
Ability to synchronize linked collection object determinations

### ISSUE THREAD

**Issue by grantfitzsimmons** [specify/specify7#2064]
> For both our wildlife and fish databases we have separate collections for the specimens and the tissue samples, and link the two collections together via the collection object relationship table (one specimen collection object can have many tissue collection objects linked to it). However the determinations of the tissue samples should always be the same as the (parent) specimen collection object - would it be possible to add an option when linking collections to allow the linked collection objects to inherit the determinations of the parent? In our case we have many tissue samples for each specimen so it would take a lot of effort to manually update.

**Requested By:** Dan Baker (@danb213) at CSIRO

**Comment by grantfitzsimmons** [comment]
@acbentley Can you share your thoughts on this issue?

**Comment by grantfitzsimmons** [comment]
Mentioned again today in the meeting with CSIRO

**Comment by acbentley** [comment]
Yes, this is something that I have been advocating for some time.  It would be great to have the ability to synchronize records that are related to ensure that redeterminations are completed in all related collections in a single action.  This could be as simple as reporting to the user any inconsistencies or as complex as automating the process of changing the value in all collections is a change is made in one - maybe somehow hooking into the batch identify tool?  I do however wonder whether we should be looking at a more holistic approach to this to be able to ensure that ALL data associated with these records is kept in sync.

**Comment by grantfitzsimmons** [comment]
Mentioned again today in the meeting with CSIRO
